buildingspy/development/regressiontest.py

```diff
@@ -2311,7 +2311,7 @@ def _funnel_plot(self, model_name, browser=None):
         server = pyfunnel.MyHTTPServer(('', 0), pyfunnel.CORSRequestHandler,
                                        str_html=content, url_html='funnel')
         # Start the browser instance.
-        server.browse(list_files, browser=browser)
+        server.browse(list_files, browser=browser, timeout=60)
```

Comment:
The default time out (if unspecified) was set to 10. I am not sure how this corrects the issue, other than it appearing after some more time.

Reply:
The timeout argument is used as a fallback to close the server, based on the following logic:
- We prioritarily check for successful HTTP requests indicating that the data to plot have been received by the browser.
- If not and if timeout is exceeded, we close the server and terminate the process that launched it.

Increasing the timeout from 10 to 60" gives more time for the server to start or for large datasets to be loaded by the browser, while still allowing manual interruption via Ctrl+C (KeyboardInterrupt).
However, as soon as successful HTTP requests, the process terminates as before.
So this should not disturb executions that were already successful, but only handle executions that are currently unsuccessful.